src/client/gui/lib/catalogue/launch_form.dart

```diff
@@ -83,6 +84,69 @@ class _LaunchFormState extends ConsumerState<LaunchForm> {
       width: 360,
     );
 
+    final zonesAsync = ref.watch(zonesProvider);
+    final zoneDropdown = FormField<String>(
+      initialValue: 'auto',
+      onSaved: (value) =>
+          launchRequest.zone = value == 'auto' ? '' : value ?? '',
+      builder: (field) {
+        return Column(
```

Comment:
Perhaps all of this layout code could be encapsulated into a separate widget so that it doesn't pollute the rest of the launch form code. Ideally it should be just a FormField that wraps a single widget that knows how to render the whole dropdown